### DISCUSSION TITLE
Unable to clone from github

### DISCUSSION THREAD

**Discussion by theaveasso** [zdharma-continuum/zinit#655] [Q&A]
When trying to install plugins from github why it ask for my github credential (username/password) instead of using ssh.

I've setup my github on my computer and it works normally. When try to don't load this `Aloxx/fzf-tab` I always get this error.
```
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/Aloxx/fzf-tab/'
```

my config
```
# zsh Plugins
zinit light zsh-users/zsh-autosuggestions
zinit light zsh-users/zsh-syntax-highlighting
zinit light zsh-users/zsh-completions
zinit light Aloxx/fzf-tab
```

**Comment by Wateir** [comment]
For everyone who have this issue
[github doc](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
No need to add any permission to the token

**Comment by pschmitt** [comment]
Or use the good ol' `insteadOf` in your git config.

https://www.omaralbeik.com/blog/enforce-ssh-instead-of-http-for-git-operations
